src/main/java/org/datadog/jenkins/plugins/datadog/DatadogUtilities.java

```diff
@@ -677,10 +677,17 @@ private static boolean isWithinIPv4OctetRange(int number) {
 
     public static Map<String, Set<String>> getComputerTags(Computer computer) {
         Set<LabelAtom> labels = null;
-        try {
-            labels = computer.getNode().getAssignedLabels();
-        } catch (NullPointerException e) {
-            logger.fine("Could not retrieve labels");
+        assert computer != null : "Computer is unexpectedly null";
```

Comment:
Could you please comment on what is the intention here? As far as I know asserts are disabled by default in Java

Reply:
I've changed the line to if (computer!=null){...}

Reply:
Could not you just do something like
```
public static Map<String, Set<String>> getComputerTags(Computer computer) {
    if (computer == null) {
        logger.fine("Could not retrieve computer tags because computer is null");
        return Collections.emptyMap();
    }
    // ...
}
```